filebeat/module/haproxy/log/test/tcplog.log-expected.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "@timestamp": "2018-09-20T15:44:23.285Z", 
+        "fileset.module": "haproxy", 
+        "fileset.name": "log", 
+        "haproxy.backend_name": "app", 
+        "haproxy.backend_queue": 0, 
+        "haproxy.bytes_read": 212, 
+        "haproxy.client.ip": "127.0.0.1", 
+        "haproxy.client.port": 40962, 
+        "haproxy.connection_wait_time_ms": -1, 
+        "haproxy.connections.active": 1, 
+        "haproxy.connections.backend": 0, 
+        "haproxy.connections.frontend": 1, 
+        "haproxy.connections.retries": 0, 
+        "haproxy.connections.server": 0, 
+        "haproxy.default.logsource": "127.0.0.1", 
```

Comment:
This logsource looks common to default and TCP, it shouldn't be under `default`, if this is always going to be an IP, it could be `haproxy.source.ip`.

Reply:
Actually, it seems it could be an IP or a hostname. I'll place this field in the root (`haproxy.logsource`) anyways.

Reply:
@ruflin what would we do ECS-wise for a source that can be an ip or a hostname?

Reply:
I don't think we have a solution here as one is ip type and other other is keyword type in ES. Probably best it to store it under keyword for now.

Reply:
@sayden umm, in haproxy documentation it says that this is an IP, also I think that source IP in default format and client IP in tcp format are the same, I'd store both in `haproxy.source.ip`.

Reply:
Well, they can also be a unix socket, tricky... @ruflin opinions for these source fields?

Reply:
`haproxy.source` then? I wouldn't use `haproxy.logsource` because it seems to be the source of the log, not the source of the logged connection.

Reply:
haproxy.source sounds like a good tradeoff for me at the moment. We should make it a keyword.